front/src/modules/activities/right-drawer/components/ActivityActionBar.tsx

```diff
@@ -22,16 +25,17 @@ type OwnProps = {
 
 export function ActivityActionBar({ activityId }: OwnProps) {
   const theme = useTheme();
-  const [createCommentMutation] = useDeleteActivityMutation();
+  const [deleteActivityMutation] = useDeleteActivityMutation();
   const [, setIsRightDrawerOpen] = useRecoilState(isRightDrawerOpenState);
 
   function deleteActivity() {
-    createCommentMutation({
```

Comment:
I don't think this was supposed to be named this way? I renamed it to match with the hook name and to its purpose